### PR TITLE
Added interface for account, and implemented it

### PR DESCRIPTION
**Pull request by antjanne**
I would love if someone saw if this was correctly implemented, I'm quite unsure. Thanks in advance!

### CODE REVIEW

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Controller/AccountController.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.goodpeople.gooddeeds.Controller;
 
 import com.goodpeople.gooddeeds.Model.Entities.Account;
```
Comment (GeneralMustard) [inline goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Controller/AccountController.java]: Remove import

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/Repositories/AccountRepository.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package com.goodpeople.gooddeeds.Model.Repositories;
 
 import com.goodpeople.gooddeeds.Model.Entities.Account;
```
Comment (GeneralMustard) [inline goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/Repositories/AccountRepository.java]: Remove import

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/Repositories/AccountRepositoryImpl.java [hunk 3]
```diff
@@ -1,6 +1,7 @@
 package com.goodpeople.gooddeeds.Model.Repositories;
 
 import com.goodpeople.gooddeeds.Model.Entities.Account;
```
Comment (GeneralMustard) [inline goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/Repositories/AccountRepositoryImpl.java]: remove import

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/Account/AccountDetails.java [hunk 4]
```diff
@@ -6,6 +6,7 @@
 import android.widget.TextView;
 
 import com.goodpeople.gooddeeds.Model.Entities.Account;
```
Comment (GeneralMustard) [inline goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/Account/AccountDetails.java]: Remove import

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Controller/AccountController.java (hunk 1 above)
Comment (antjanne) [reply goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Controller/AccountController.java]: Fixed

File: goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/GoodDeeds.java [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 import com.goodpeople.gooddeeds.Model.Entities.Account;
```
Comment (Rantaa) [inline goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/Model/GoodDeeds.java]: Remove import

**Pull request merged**